pages/common/lein.md

```diff
@@ -0,0 +1,23 @@
+# lein
+
+> Manage clojure projects with declarative configuration.
+
+- Generate scaffolding for a new project based on a template:
+
+`lein new {{template}} {{project_name}}`
+
+- Start a repl session either with the project or standalone:
+
+`lein repl`
+
+- Run the project's -main function with optional args:
+
+`lein run {{args}}`
+
+- Run the project's tests:
+
+`lein test`
+
+- Package up the project files and all dependencies into a jar file:
```

Comment:
A minor readability tweak, but perhaps `and all its dependencies` would sound better?